extensions/Worker.Extensions.Tables/src/Worker.Extensions.Tables.csproj

```diff
@@ -12,16 +12,13 @@
 
   <Import Project="..\..\..\build\Extensions.props" />
 
-  <ItemGroup>
-    <ProjectReference Include="..\..\Worker.Extensions.Abstractions\src\Worker.Extensions.Abstractions.csproj" />
-    <ProjectReference Include="..\..\..\src\DotNetWorker.Core\DotNetWorker.Core.csproj" />
-  </ItemGroup>
-
   <ItemGroup>
     <PackageReference Include="Azure.Data.Tables" Version="12.8.3" />
     <PackageReference Include="Microsoft.Extensions.Azure" Version="1.7.4" />
     <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="7.0.0" />
     <PackageReference Include="System.Linq.Async" Version="6.0.1" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker.Core" Version="1.19.0" />
+    <PackageReference Include="Microsoft.Azure.Functions.Worker.Extensions.Abstractions" Version="1.3.0" />
```

Comment:
See comment for blob extension - most of the extensions kept Abstractions as a project reference.

Reply:
Commented above.